pom.xml

```diff
@@ -21,6 +21,16 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-web</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.logging.log4j</groupId>
```

Comment:
duplicate

Reply:
done

Reply:
Please next time shorten the diffs in the PR when you perform a similar operation like this.
Removing, the other log4j2 dependency declaration increases the diffs to review.